src/libs/actions/IOU.js

```diff
@@ -2658,30 +2667,42 @@ function approveMoneyRequest(expenseReport) {
  */
 function submitReport(expenseReport) {
     const optimisticSubmittedReportAction = ReportUtils.buildOptimisticSubmittedReportAction(expenseReport.total, expenseReport.currency, expenseReport.reportID);
+    const parentReport = allReports[`${ONYXKEYS.COLLECTION.REPORT}${expenseReport.parentReportID}`];
```

Comment:
I think we should consider a default value as an empty object.

```suggestion
    const parentReport = allReports[`${ONYXKEYS.COLLECTION.REPORT}${expenseReport.parentReportID}`] ?? { };
```